modules/end-user-guide/partials/proc_building-custom-intellij-editor-images.adoc

```diff
@@ -0,0 +1,151 @@
+[id="building-custom-intellij-editor-images_{context}"]
+= Building custom IntelliJ editor images
+
+This procedure describes how to build an IntelliJ-based editor image using a custom archive and a custom image name.
+
+.Prerequisites
+
+* Define the following locations to publish the editor image and metadata:
++
+`__<IntelliJ-container-image>__`:: Location of the IntelliJ editor image in a registry accessible by {prod-short}.
++
+`__<URL_to_meta.yaml>__`:: HTTPS resource containing the editor metadata.
+
+* Define the build parameters:
++
+`--tag __<name:tag>__`::
+Tag to apply to the image after build.
++
+`--url __<downloadUrl>__`::
+Download URL pointing to the archive of the custom IntelliJ-based distribution, meeting following criteria: 
```

Comment:
@themr0c "Download URL" - meaning how? On the command line? Or just copy-paste?

Reply:
@themr0c Also, what's the purpose of the URL or where will it be used?

Reply:
@themr0c Or does it mean that the user has to download the archive using that URL?